### ISSUE TITLE
Language toggle link missing lang attribute

### ISSUE THREAD

**Issue by mnigh** [GCTC-NTGC/gc-digital-talent#2964]
## Description

> Any content in a language other than the page language must be marked up accordingly. For the language toggle link on the top right, its code should be as follows:

`<a href="/fr" lang="fr">Français</a>`
